source/reference/method/MongoDBCollection-find.txt

```diff
@@ -52,9 +52,15 @@ Parameters
 
      * - batchSize
        - integer
-       - The number of documents to return in the first batch. Defaults to
-         ``101``. A batchSize of ``0`` means that the cursor will be
-         established, but no documents will be returned in the first batch.
+       - The maximum number of documents to return in the first batch. By default, the ``find()`` 
```

Comment:
Not sure why this leading sentence differs so much from the others. Also, the command should be written as `find`. `find()` would refer to the helper method in the driver.